_posts/2020-12-02-welcome-new-hire-getting-started-in-frontend-at-rimdev-updated.md

```diff
@@ -0,0 +1,93 @@
+
+
+## Communicating
+
+Communication is at our core; we can accomplish more as our sum that we can as our parts, so we talk. About projects, goals, vision, and the path to get there. Slack is where we live, comment, post [gifs](https://giphy.com), [cat videos](https://www.youtube.com/watch?v=5dsGWM5XGdg), ya know.
+
+- [Slack](https://slack.com/)! We live here
+- [MS Teams](https://products.office.com/en-us/microsoft-teams/group-chat-software). Chat with non-slackers.
+- Outlook
+
+Looking to get up to speed on either? Here are some quick keyboard references for [Slack](https://slack.com/help/articles/201374536-Slack-keyboard-shortcuts) and [Teams](https://support.microsoft.com/en-us/office/keyboard-shortcuts-for-microsoft-teams-2e8e2a70-e8d8-4a19-949b-4c36dd5292d2.)
+
+## Environment
+
+RIMdev is always on a journey to deliver a great, and constantly improving, frontend experience for our users. We leverage [VueJS](https://vuejs.org/), [webpack](https://webpack.js.org/) (although [ParcelJS](https://parceljs.org/) is gaining ground🤞), and [Sass](https://sass-lang.com/) heavily to build our Ritter Platform and family of static marketing sites. We've also built our own CSS framework [Platform UI](https://style.rimdev.io/) which continues to grow and be applied across our ecosystem. If you want to take a closer look, head over to [npm](https://www.npmjs.com/package/@ritterim/platform-ui) and grab a copy.
+
+- [Node](https://nodejs.org/en/download) & [npm](https://www.npmjs.com/)
+- [Homebrew](https://docs.brew.sh/Installation)🍺: Mac's missing package manager.
+- [Git](https://www.git-scm.com/) & [GitHub](https://github.com)
+- [RBenv](https://github.com/rbenv/rbenv): Ruby version management (for [Jekyll](https://jekyllrb.com/))
+- [Jekyll](https://jekyllrb.com/): Ruby-based static site generator.
+- [Hugo](https://gohugo.io/getting-started/quick-start/): [Go](https://golang.org/) based static site generator.
+- [VS Code](Visual Studio Code - Code Editing. Redefined
+https://code.visualstudio.com/), [Atom](https://atom.io/), other?
+- [iTerm2](https://www.iterm2.com/) or "[Oh My ZSH!](https://ohmyz.sh/)"
+- [.NET SDK](https://dotnet.microsoft.com/download)
+- [Docker Desktop](https://www.docker.com/products/docker-desktop): To run our shared dependencies locally (MSSQL, Elasticsearch, Azurite).
+
+## UI/UX Design and prototyping
+
+All things [Adobe](https://www.adobe.com/), but especially [XD](https://www.adobe.com/products/xd.html)!
+
+## Supporting cast
+
+- [LastPass](https://www.lastpass.com/)
+- [Highfive](https://highfive.com): Video conferencing.
+- [Spotify](https://spotify.com) Premium 🎧
+- [Headphones](https://www.cnet.com/topics/headphones/products/)
+
+## More online 
+
+- [Codepen](https://codepen.io): A great place to work out problems and share! Create an account now! (shameless plug!)
```

Comment:
- add code
[Code Sandbox](https://codesandbox.io/): Great place to prototype Vue Apps.
[Vue Telemetry](https://vuetelemetry.com/): Tool to find websites made with Vue.